cloudinit/config/cc_ntp.py

```diff
@@ -23,25 +23,7 @@
 frequency = PER_INSTANCE
 NTP_CONF = "/etc/ntp.conf"
 NR_POOL_SERVERS = 4
-distros = [
-    "almalinux",
-    "alpine",
-    "centos",
-    "cloudlinux",
-    "debian",
-    "eurolinux",
-    "fedora",
-    "miraclelinux",
-    "openEuler",
-    "openmandriva",
-    "opensuse",
-    "photon",
-    "rhel",
-    "rocky",
-    "sles",
-    "ubuntu",
-    "virtuozzo",
-]
+distros = ["all"]
```

Comment:
I don't think we can assume this. E.g., Arch and Gentoo aren't in the original list, and so this module probably won't work on them.

Reply:
oy! thanks for the catch. i did not notice that